dynasm/dynasm.lua

```diff
@@ -861,6 +861,10 @@ local function doline(line)
   -- Strip assembler comments.
   aline = gsub(aline, "//.*$", "")
 
+  -- Process and remove labels.
+  for stmt in gmatch(aline, "[^:]+:") do dostmt(stmt) end
```

Comment:
I just realized it's possible to use the ternary operator in statements, as in: `| push (arg ? 1 : 0)`, so this regex won't work.